clvm/SExp.py

```diff
@@ -1,9 +1,10 @@
 import io
```

Comment:
We're not doing `from __future__ import annotations` in this file?

Reply:
Most of that stuff has only really been set up in blockchain, yeah.  But we could either add the tooling or just manually add the import here.

Reply:
Yeah my point was to add it, not about the tooling.  There's lots of "" in the file is why.

Reply:
[`7222f09` (#125)](https://github.com/Chia-Network/clvm/pull/125/commits/7222f093be62676bf1ce1de2059985b7bc11fdf5)

Reply:
though i got a bit overzealous and had to roll some back in [`23e47d8` (#125)](https://github.com/Chia-Network/clvm/pull/125/commits/23e47d85183139b8b6e17f2831040a40bcc7c5cd)